packages/dds/tree/package.json

```diff
@@ -66,6 +66,7 @@
     "temp-directory": "nyc/.nyc_output"
   },
   "dependencies": {
+    "@fluidframework/common-definitions": "^0.20.1",
```

Comment:
I think this line isn't supposed to come with the cherry pick?

Reply:
Hmmm, I saw a lint failure at some point, because the package is used in a test file here: https://github.com/microsoft/FluidFramework/blob/9dc84af95021514c509377e4c4d0d4726c9f7de6/packages/dds/tree/src/test/shared-tree-core/sharedTreeCore.spec.ts#L6

That said, I was playing around with pnpm at the time too so my dependencies might have been in an unusual state. Let me revert this and do a fresh build.

Reply:
Yeah, I get a comnsistent lint error from import/no-extraneous-dependencies. I moved it to devDeps, though, since it's only used in a test file.

Reply:
Ah, worth pointing out that in main we [left it in `dependencies` and removed it from `devDependencies`](https://github.com/microsoft/FluidFramework/pull/13029/files#diff-34daa9b8b6fa1ce21c7854703df1f12799bbc4aa32e0bc8363418bc9df10a356). Not sure if its use was expanded outside the tests in that branch.

Reply:
Hmmm, probably safest to do what we did in main. Thanks for pointing that out. I'll move it back.